catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/TestNsiliEndpoint.java

```diff
@@ -134,20 +138,51 @@ public void testStandingQueryMgr() throws Exception {
     @Test
     public void testMaxNumResults() throws Exception {
         int currMaxNum = nsiliEndpoint.getMaxNumResults();
+
         nsiliEndpoint.setMaxNumResults(100);
         int setNumResults = nsiliEndpoint.getMaxNumResults();
         assertThat(setNumResults, is(100));
+
         nsiliEndpoint.setMaxNumResults(currMaxNum);
     }
 
     @Test
-    public void testQuerySources() throws Exception {
-        String testSource = "source1";
-        List<String> existingSources = nsiliEndpoint.getQuerySources();
-        List<String> querySources = Arrays.asList(new String[] {testSource});
-        nsiliEndpoint.setQuerySources(querySources);
-        List<String> setSources = nsiliEndpoint.getQuerySources();
-        assertThat(setSources, hasItem(testSource));
+    public void testSetQuerySources() throws Exception {
+        Set<String> existingSources = new HashSet<>(nsiliEndpoint.getQuerySources());
+
+        Set<String> testSources = new HashSet<>();
+        testSources.add(TEST_SOURCE_ID);
+        nsiliEndpoint.setQuerySources(testSources);
+        assertThat(nsiliEndpoint.getQuerySources(), equalTo(testSources));
+
+        nsiliEndpoint.setQuerySources(existingSources);
```

Comment:
The already-existing tests  `testMaxNumResults()` and `testQuerySources()` reset the maxNumResults and querySources, respectively, after testing.